### PR TITLE
Add nanoflow event

### PR DESCRIPTION
**Pull request by McDoyen**

### CODE REVIEW

File: src/RichText.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
...JREFUeAHtW72OEzEQ3g1IQJn+gI4WpGsQT0AaREESeIGrrsxrnES9DbRJQDrRHC0010JDjY68ANLBQZPc571dnT32xo4zXhzWadZjz+839uyPnSxLv4RAQqDLCOQ0+OFweAN9h3meP8FVtG2/89Vq9WY+n3+yMXKOj8fjwXK5LKBzz1HvotfrHUyn0xOZ/6ZMVO1DXF8jKMNQY9dLAPcAIJw1cjAPbBi8sL5XydyVXenJhGhXmafdNvoWGPZtTMzjrpmXzWoyGgDgdpn2stKyDeBMujS+2DpMS0DxEYEdoeNU6bwiJlgmj+t+tO9jXT6s6dBXTGfFBJafVs8EA5bm2rVsBQA6Tmez2XvFGojRaPSK9B1Rp8h4lOROTltOJDsPgMsScAIcteI7GH86MTMwoeaw1Bs2ABDTxFQrGGI1qrAVN6OQodMFgImh4GXyHcCgt/UuX0CsAMQWKDeypiJ47mMEQP3ykdtCZuEhq8loAOCF4S0U/91Q+bd+v/95Q5mt2MWLDRRoAa1RWr4M0fGmp6d7YNyXH2+R4XeyMMZeCFpkXgRfFMVvQQ8+/Blky1WBxy/tuVuM+/7g6CLr5Qcnz24rb3O1PloDmp4Ma/76aqwB1VvdWc0krjAgk1ljxQ8QvDAsAM2hG03lbU5xyoPQloCHDkWEO/Oy8hC62QGQHd6FtnEJcDr+8fkdY515enyBhF7/XPmuJXhanZ8B7ACU1bpKjtz2zZesQ2776qNy7ACIW5VwtHQWbWpwY5pbH3GAvQZU92m2WxW3PhJ/xj8DqIXI6QRA5AkK7l6aAcEhjtxAmgGRJyi4e2kGBIc4cgNpBkSeoODupRkQHOLIDaQZEHmCgrvX+gwQ+waD44sfNDLRV+4p0IHAdOsANG2alJ+8r777Bw5ZVd8+AKr9f061D0D1jY9GzvYNkSq20LCr/rhOYKpa+SkuP7UZsMUJTP4o12jk8lMDADZ9dnV9ZNaE5zTkY1OTYfssjmM0X5zcZmLCtjyLJisATfvsdD8eDrGc2vKNytVPqt+0BCjPf013HgDrEnBNP87sPHLl5eDDXYCl5rABgH9ifOUIzFUHPbLjKkf50hKgiFCaVns6Hgvt62fnZ4AJgIVHVn1kPMwoIj42NRkNAK4TmIqrAYhd8TNA6EllQiAhwIjAJSEj9lh5/+kBAAAAAElFTkSuQmCC</icon>
```
Comment (jseremba) [inline src/RichText.xml:5]: Invalid icon, use
`iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAMAAACdt4HsAAAAVFBMVEUAAABVVVVmZmZVVVVXV1dVVVVYWFhVVVVJsvhXV1dVVVVWVlZVVVVVVVVWVlZWVlZWVlZWVlZWVlZWVlZVVVVWVlZJsPdVVVVVVVVVVVVIsPdVVVVYxw9VAAAAGnRSTlMAAwoeLz9ASElJUVNvcoygqazH0NXf4+Pk+DNXn6MAAADwSURBVHja7ZXrDoIwDEanoAyGctOi7P3fU8C4suA65gxq3PnVfKEnbJSUBe5s8rJ6UOwYQdpKRZuqOJcTrhEhGPrRoOJSTskIgdRQcaXF4jXBUQzUfXWIzWDbTCBURfEjgnNjxi6wfAVCUFcDFxSsOgeFFifuo7y/SuS0df+ZWJQJIcbXF0nfz6F7AnA8huWGsV83LBZ0BtYVzKpVBaAuDNvA5RI5AEeBllkFCAoUQRAEQfC/Ag64D9wFuB/gIwL7Eewr075ciaXtuN6lBiWQGoa4ISAEWFHgk18tiAkWCRjBmwW+c+Av8B5l/58pMHID+k6HUjc3bmAAAAAASUVORK5CYII=`

File: src/RichText.xml [hunk 2]
```diff
@@ -106,5 +106,11 @@ WARNING disabling this option is a security risk for XSS attacks</description>
             <description>Microflow will be triggered when focus is lost and the content has changed</description>
             <returnType type="Void"/>
         </property>
+        <property key="onChangeNanoflow" type="nanoflow" required="false">
+            <caption>On change nanoflow</caption>
+            <category>Events</category>
+            <description>Nanoflow will be triggered when focus is lost and the content has changed</description>
```
Comment (jseremba) [inline src/RichText.xml:112]: i think description should be 
`Nanoflow that will be triggered when focus is lost and the content has changed` and this applies to the microflow

File: src/components/RichTextContainer.ts [hunk 3]
```diff
@@ -150,15 +155,28 @@ export default class RichTextContainer extends Component<RichTextContainerProps,
...
+                error: error => window.mx.ui.error(`Error while executing microflow: ${onChangeMicroflow}: ${error.message}`)
+            });
+        }
+
+        if (onChangeNanoflow.nanoflow) {
+            const context = new mendix.lib.MxContext();
+            context.setContext(mxObject.getEntity(), mxObject.getGuid());
+            window.mx.data.callNanoflow({
+                context,
+                error: error => window.mx.ui.error(`Error while executing nanoflow: + ${onChangeNanoflow} : ${error.message}`),
```
Comment (jseremba) [inline src/components/RichTextContainer.ts]: Re-format error to
`window.mx.ui.error(`Error while executing nanoflow: ${onChangeNanoflow} : ${error.message}``
Plus sign not required for template strings

**Review by jseremba — CHANGES_REQUESTED** [review]

**Comment by Andries-Smit** [comment]
Merge via upstream

**Pull request closed without merging**